src/main/java/ejercicio/banco/util/DataBaseUtil.java

```diff
@@ -0,0 +1,93 @@
+package ejercicio.banco.util;
+
+import java.sql.*;
+
+public class DataBaseUtil {
+
+    public static void main(String[] args) {
+        Connection con = startConnection("bank_project");
+        executeQuery(con, " imposible query");
+        try {
+            showTables(con);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        closeConections(con);
+    }
+
+    public static Connection startConnection(String tablename) {
+        Connection con = null;
+        try {
+            con = DriverManager
+                    .getConnection("jdbc:mysql://localhost:3306/"+tablename, "user", "user");
```

Comment:
This is wrong. The JDBC connector doesn't require the table name but the database name (`bank_project`in our case).

Reply:
changed name of the variable to avoid confusion